packages/toolpad-app/src/toolpad/AppEditor/PageEditor/ComponentCatalog/ComponentIcon.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import dynamic from 'next/dynamic';
+
+const SmartButtonIcon = dynamic(() => import('@mui/icons-material/SmartButton'));
```

Comment:
Isn't lazy loading these a bit overkill? This will create a bundle for each and every one of these icons. I wouldn't be surprised if that will create more overhead than just importing them.

Aside for the future, I also want to avoid using `next/dynamic` when `React.lazy` works well enough.